report/3.Theory/2.RTS.tex

```diff
@@ -44,4 +44,42 @@ \subsubsection{Task-based scheduling}
 
 This scheduling method can work both with and without pre-emption of tasks, meaning that a running task can be set on hold when a higher priority task is ready to run.
 When using this method for scheduling, a task with a low priority is allowed to run if no other task with a higher priority is ready to run. 
-Given a low priority task is running and a higher priority task is marked as ready, the low priority task will be pre-empted in order to allow the higher priority to run.
\ No newline at end of file
+Given a low priority task is running and a higher priority task is marked as ready, the low priority task will be pre-empted in order to allow the higher priority to run.
```

Comment:
Isn't it: "Given that" Syntes i hvertfald at der er et eller andet galt i begyndelsen af sætningen